functests/4_latency/latency.go

```diff
@@ -42,7 +42,7 @@ const (
 	defaultTestRuntime   = "300"
 	defaultMaxLatency    = -1
 	defaultTestCpus      = -1
-	minCpuAmountForOslat = 2
+	minCpuAmountForOslat = 3
```

Comment:
3 is an odd number this will result with SMTAlignment error in pods events, to avoid that we need an even number so it'll start at 4.

Reply:
please ignore above comment, I am mistaken here, as this only indicates the minimal value and doesn't apply it anywhere

Reply:
This is also a tricky check, meaning do we want to ensure that an even number of CPUs is about to be tested?

Reply:
yes, we want to allow tests with even cpus number only, otherwise the test pod will not start because of the SMTAlignment error

Reply:
But what if SMT alignment is disabled?

Reply:
is that an option? HT is supported since 4.10 and I guess should also be spilled to 4.11